ebean-migration/src/main/java/io/ebean/migration/runner/MigrationTable.java

```diff
@@ -501,6 +526,10 @@ List<MigrationResource> runAll(List<LocalMigrationResource> localVersions) throw
         break;
       }
     }
+    if (patchLegacyChecksums && !checkStateOnly) {
+      // only patch the legacy checksums once
+      initMetaRow.resetChecksum(EARLY_MODE_CHECKSUM, connection, updateChecksumSql);
```

Comment:
CHECKME: Could this affect logicalLock? Or should this be done at the end, after all migrations are ran?

Reply:
It's all good with the logical lock, different columns being updated.  Note that the tests use H2 which uses the logicalLock. So yes, I'm happy that we are good with this.